src/index.js

```diff
@@ -0,0 +1,653 @@
+            let value = table.rows[row].cells[cell].children[0].value;
+            let number = parseFloat(value);
+            if (isNaN(number)) {
+                return [];
+            }
+            array.push(number);
+        }
+    }
+    return array;
+}
+
+function setCellValues(array) {
+    const table = document.getElementById('array-table');
+    for (let row = 0; row < table.rows.length; row++) {
+        for (let cell = 0; cell < table.rows[row].cells.length; cell++) {
+            let child = table.rows[row].cells[cell].children[0];
+            child.value = array[row * table.rows[0].cells.length + cell];
+        }
+    }
+    return array;
+}
+
+function onArrayCommandClick(element) {
+    let array = getCellValues();
+    if (array.length > 0) {
+        let dest = document.getElementById('array-result');
+        switch (element.value) {
+            case 'Sub Sum':
+                dest.value = ArrayTool.getMaxSubSumFast(array);
+                break;
+            case 'Sub Sum Slow':
+                dest.value = ArrayTool.getMaxSubSumSlow(array);
+                break;
+            case 'Min':
+                dest.value = ArrayTool.getMin(array);
+                break;
+            case 'Max':
+                dest.value = ArrayTool.getMax(array);
+                break;
+            case 'Median':
+                dest.value = ArrayTool.getMedian(array);
+                break;
+            case 'Selection':
+                dest.value = ArrayTool.getIncSequence(array);
+                break;
+            case 'Heap sort':
+                setCellValues(ArraySorter.heapSort(array));
+                break;
+            case 'Quick sort':
+                setCellValues(ArraySorter.quickSort(array));
+                break;
+            case 'Counting sort':
+                setCellValues(ArraySorter.countingSort(array));
+                break;
+            case 'Insertion sort':
+                setCellValues(ArraySorter.insertionSort(array));
+                break;
+        }
+    }
+}
+
+function onTextFormatClick() {
+    let formatValue = document.querySelector('input[name="wrap"]:checked').value;
+    let formatType = formatValue === 'noWrap' ? FormatType.noWrap :
+        formatValue === 'wordWrap' ? FormatType.wordWrap :
+            formatValue === 'sentenceWrap' ? FormatType.sentenceWrap : FormatType.charWrap;
+    let source = document.getElementById("formatter-input").value;
+    let dest = document.getElementById("formatter-output");
+    const maxLength = parseInt(document.getElementById("formatter-maxlength").value);
+    const maxStrings = parseInt(document.getElementById("formatter-maxstrings").value);
+    if (isNaN(maxLength) || isNaN(maxStrings)) {
+        dest.value = source;
+    } else {
+        dest.value = TextFormatter.format(source, maxLength, maxStrings, formatType);
+    }
+}
+
+function onCalculateClick() {
+    const allowedSymbols = '0123456789.+-*/()'
```

Comment:
better to move out of function scope so now it is created every time the function is called.